liveHomeFrontend/src/components/Map.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from '../assets/styles/components/Map.module.sass';
+import Button from './Button';
+
+const Map = ({ children }) => {
+  return (
+    <div className={styles.mapContainer}>
+      {children}
```

Comment:
This component receive the children, but i think it should render the Google Map directly

Reply:
i think that this component it's ok, because it's a layout for google maps functionality